src/main/java/com/tibagni/logviewer/LogViewerPresenter.java

```diff
@@ -133,6 +133,15 @@ public void removeFilters(String group, int[] indices) {
     }
   }
 
+  @Override
+  public void removeGroup(String group){
+    if(!StringUtils.isEmpty(group)){
+      filters.remove(group);
```

Comment:
Before actually removing the filter, you can check if it has been changed by the user ('haveFiltersChanged') and, if so, as if the user wants to save it before removing.
Check example on 'finishing'

Reply:
Done